src/controllers/users.js

```diff
@@ -0,0 +1,82 @@
+'use strict';
+
+const userService = require('../services/users');
+
+const getAll = (req, res) => {
+  const users = userService.getAll();
+
+  res.send(users);
+};
+
+const getOne = (req, res) => {
+  const { userId } = req.params;
+
+  const currentUser = userService.getById(userId);
+
+  if (!currentUser) {
+    res.sendStatus(404);
+
+    return;
+  }
+
+  res.send(currentUser);
+};
+
+const add = (req, res) => {
+  const { name } = req.body;
+
+  if (!name || typeof name !== 'string') {
```

Comment:
if you have several checks will be better to move the result to separate boolean variable